scalyr_agent/builtin_monitors/kubernetes_events_monitor.py

```diff
@@ -526,6 +506,25 @@ def run(self):
             global_log.info('kubernetes_events_monitor exiting because it has been disabled.')
             return
 
+        def _create_k8s_api(rate_limiter_key):
```

Comment:
This function seems to rely on several variables being declared in the outer scope, e.g. `k8s_verify_api_queries` and `k8s_api_url`.

I wonder if these should be passed as parameters instead.  Otherwise non-obvious problems will likely occur if this is ever refactored out in to a class-level function rather than a nested function.